web/components/VenueNav/VenueNav.tsx

```diff
@@ -0,0 +1,50 @@
+import clsx from 'clsx';
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+import urlJoin from 'proper-url-join';
+import type { Venue } from '../../types/Venue';
+import styles from './VenueNav.module.css';
+
+interface VenueNavProps {
+  venues: Venue[];
+  activeVenue?: Venue;
+}
+
+const venueLabel = (venueName: string) =>
+  ({
+    'San Francisco': 'Main venue',
+    Virtual: 'Online',
+  }[venueName] || 'Satellite');
```

Comment:
Is there a less hacky way to classify the `'Main venue'` here?

Previously we've looked at the first venue in the event document, I think, but I suppose another approach could be to hard-code a `mainVenueId` in a similar way to the existing `mainEventId`. Don't know which is easier.

Reply:
What do you think of 60569da? It's the best I could come up with given the circumstances.